paml_convert/autoprotocol/transcriptic_api.py

```diff
@@ -0,0 +1,269 @@
+import json
+import os
+import requests
+import time
+import transcriptic
+
+from datetime import datetime
+import requests
+from requests_html import HTMLSession
+
+
+import logging
+
+l = logging.getLogger(__file__)
+l.setLevel(logging.ERROR)
+
+class TranscripticException(Exception):
```

Comment:
Shouldn't this be Strateos? Or is that an upstream issue?
We should at least have a comment about Trancriptic vs. Strateos

Reply:
Their API is called transcriptic.py.  It is largely historical.   I can rename all our stuff for Strateos, but continue to use their transcriptic package.